sdk/services/futures.ts

```diff
@@ -272,6 +272,32 @@ export default class FuturesService {
 		return positions;
 	}
 
+	public async getMarketFundingRates(marketAsset: FuturesMarketAsset) {
```

Comment:
Could we rename this to `getMarketFundingRatesHistory` just so it's more clear this is for historical data rather than current rates.

Reply:
Good idea. Added in the latest commit.